Application/Frontend/components/FriendsColumn/FriendsTab.tsx

```diff
@@ -49,20 +55,18 @@ export function FriendsTab(props: TextInputProps) {
        */
       useEffect(() => {
         if (user) { // IMPORTANT: There is a slight delay in the user object being available after login, so we need to wait for it to not be null
-          console.log(user.id);
           const fetchData = async () => {
             try {
```

Comment:
- Error handling for API requests could be improved by adding a catch block to handle errors and providing appropriate feedback to the user.
- It might be beneficial to add PropTypes or TypeScript types/interfaces for the props passed to the component to ensure type safety and better documentation.